File: MPP_Proiect/MPP_Proiect/domain/Angajat.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MPP_Proiect.domain
{
    class Angajat : Entity<long>
    {
        private string nume;
        private string prenume;
        private string mail;
        private string password;

        public Angajat(string nume, string prenume, string mail, string password)
        {
            this.nume = nume;
            this.prenume = prenume;
            this.mail = mail;
            this.password = password;
        }

        public string getNume()
        {
            return nume;
        }

        public void setNume(string nume)
        {
            this.nume = nume;
        }

        public string getPrenume()
        {
            return prenume;
        }

        public void setPrenume(string prenume)
        {
            this.prenume = prenume;
        }

        public string getMail()
        {
            return mail;
        }

        public void setMail(string mail)
        {
            this.mail = mail;
        }

        public string getPassword()
        {
            return password;
        }

        public void setPassword(string password)
        {
            this.password = password;
        }


        public override string ToString()
        {
            return "Angajat{" +
                "id=" + id +
                ", nume='" + nume + '\'' +
                ", prenume='" + prenume + '\'' +
                ", mail='" + mail + '\'' +
                ", password='" + password + '\'' +
                '}';
        }

    }
}

File: MPP_Lab_Proiect/src/domain/Angajat.java
package domain;

public class Angajat extends Entity<Long> {
    private String nume;
    private String prenume;
    private String mail;
    private String password;


    public Angajat(String nume, String prenume, String mail, String password) {
        this.nume = nume;
        this.prenume = prenume;
        this.mail = mail;
        this.password = password;
    }

    public String getNume() {
        return nume;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }

    public String getPrenume() {
        return prenume;
    }

    public void setPrenume(String prenume) {
        this.prenume = prenume;
    }

    public String getMail() {
        return mail;
    }

    public void setMail(String mail) {
        this.mail = mail;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    @Override
    public String toString() {
        return "Angajat{" +
                "id=" + id +
                ", nume='" + nume + '\'' +
                ", prenume='" + prenume + '\'' +
                ", mail='" + mail + '\'' +
                ", password='" + password + '\'' +
                '}';
    }
}
